var/spack/repos/builtin/packages/libxcvt/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class Libxcvt(MesonPackage):
+    """Implementation of the VESA CVT standard timing modelines generator."""
+
+    homepage = "https://gitlab.freedesktop.org/xorg/lib/libxcvt"
+    url = "https://gitlab.freedesktop.org/xorg/lib/libxcvt/-/archive/libxcvt-0.1.2/libxcvt-libxcvt-0.1.2.tar.bz2"
```

Comment:
Almost all xorg packages used `XorgPackage` and `xorg_mirror_path` here. Can you update it to use that as well?